includes/REST_API/Link_Controller.php

```diff
@@ -150,7 +146,7 @@ public function parse_link( $request ) {
 
 		if ( ! $html ) {
 			set_transient( $cache_key, wp_json_encode( $data ), $cache_ttl );
```

Comment:
Is this was an error. Why is it being cached?

Reply:
An oversight.

```suggestion
```

Reply:
If the HTML of the page is malformed and cannot be parsed, it is very likely that this isn't a once-off issue, but a persistent one. That's why the caching makes sense here and further below.